docs/reST/ref/code_examples/cursors_module_example.py

```diff
@@ -0,0 +1,42 @@
+# pygame setup
+import pygame
+
+pygame.init()
+screen = pygame.display.set_mode([600, 400])
+pygame.display.set_caption("Example code for the cursors module")
+
+# create a system cursor
+system = pygame.cursors.Cursor(pygame.SYSTEM_CURSOR_NO)
+
+# create bitmap cursors
+bitmap_1 = pygame.cursors.Cursor(*pygame.cursors.arrow)
+bitmap_2 = pygame.cursors.Cursor(
+    (24, 24), (0, 0), *pygame.cursors.compile(pygame.cursors.thickarrow_strings)
+)
+
+# create a color cursor
+surf = pygame.Surface((40, 40)) # you could also load an image 
+surf.fill((120, 50, 50))        # and use that as your surface
+color = pygame.cursors.Cursor((20, 20), surf)
+
+cursors = [system, bitmap_1, bitmap_2, color]
+cursor_index = 0
+
+pygame.mouse.set_cursor(cursors[cursor_index])
+
+clock = pygame.time.Clock()
+while True:
```

Comment:
I've been trying to get all the examples to use pg, and quit in the following way...
```python
import pygame as pg
...
going = True
while going:
    for event in pygame.event.get():
        if event.type == pygame.QUIT or (event.type == pg.KEYDOWN and e.key == K_ESCAPE):
            going = False
```

Reply:
Made a little edit your comment illume, if you don’t mind :)